apps/console/src/features/authentication/store/actions/authenticate.ts

```diff
@@ -48,6 +48,8 @@ export const getProfileInformation = (
 
     dispatch(setProfileInfoRequestLoadingStatus(true));
 
+    const isSubOrg: boolean = window[ "AppUtils" ].getConfig().organizationName;
```

Comment:
shall we use this instead? https://github.com/wso2/identity-apps/blob/1874a957cdc2f704d6514d5eed8d7b64940514e1/apps/console/src/features/organizations/hooks/use-get-organization-type.tsx#L81

Reply:
We will have to use a useEffect hook and sent the isSubOrganization() value from the use-sign-in.ts page..